core/space/services/services_fs.go

```diff
@@ -548,6 +548,15 @@ func (s *Space) addFile(ctx context.Context, sourcePath string, targetPath strin
 			return domain.AddItemResult{}, err
 		}
 
+		//add current user as admin on the remote file
+		err = s.tc.AddCurrenUserAsFileOwner(ctx, b, targetPathBucket)
+		if err != nil {
+			log.Error("Error adding current user as file owner", err)
```

Comment:
@jsonsivar 

It looks like a good candidate for _optional_ error tracking (eg: [Sentry](https://docs.sentry.io/platforms/go/)) in the future.